examples/todoapp/buildSrc/buildSrc/build.gradle.kts

```diff
@@ -2,6 +2,11 @@ plugins {
     `kotlin-dsl`
 }
 
-repositories {
-    jcenter()
+repositories.jcenter()
+
+sourceSets.main {
+    java {
```

Comment:
Why `java`?

Reply:
I can't get this behavior using the `kotlin` block and searching on Google I only see implementations using the `java` block.